packages/flutter/test/widgets/selectable_text_test.dart

```diff
@@ -176,6 +176,43 @@ void main() {
     );
   }
 
+  testWidgets('throw if no Overlay widget exists above', (WidgetTester tester) async {
+    final FlutterExceptionHandler? handler = FlutterError.onError;
+    final List<FlutterErrorDetails> errors = <FlutterErrorDetails>[];
+    FlutterError.onError = (FlutterErrorDetails details) {
+      errors.add(details);
+    };
+
+    await tester.pumpWidget(
+      const Directionality(
+        textDirection: TextDirection.ltr,
+        child: MediaQuery(
+          data: MediaQueryData(size: Size(800.0, 600.0)),
+          child: Center(
+            child: Material(
+              child: SelectableText('I love Flutter!'),
+            ),
+          ),
+        ),
+      ),
+    );
+
+    final Offset textFieldStart = tester.getTopLeft(find.byType(SelectableText));
+    final TestGesture gesture = await tester.startGesture(textFieldStart, kind: PointerDeviceKind.mouse);
+    await tester.pump(const Duration(seconds: 2));
+    await gesture.up();
+    await tester.pumpAndSettle();
+
+    await tester.pumpWidget(const SizedBox.shrink());
+    FlutterError.onError = handler;
```

Comment:
could you get that error via `tester.takeException` instead of overriding the handler?

Reply:
done